fallout/jobs/job_types/legion/veteranlegionary.dm

```diff
@@ -0,0 +1,29 @@
+/datum/job/f13/legion/veteranlegionary
+	title = "Veteran Legionary"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "The Centurion, Veteran Decanii."
+	description = "Fight in the name of Caesar, and claim the lands for the bull! Listen to your superiors, They will guide you to victory."
+
+	outfit = /datum/outfit/job/f13/legion/veteranlegionary
+
+	display_order = JOB_DISPLAY_ORDER_F13_VETERANLEGIONARY
+
+/datum/outfit/job/f13/legion/veteranlegionary
+	name = "_Legion Veteran Legionary"
+	jobtype = /datum/job/f13/legion/veteranlegionary
+
+	head = /obj/item/clothing/head/helmet/fallout/legion/veteran
+	glasses = /obj/item/clothing/glasses/sunglasses/big
+	back = /obj/item/storage/backpack/satchel/leather
+	uniform = /obj/item/clothing/under/fallout/legion/fatigues/red
+	shoes = /obj/item/clothing/shoes/fallout/military/legion/darkboots
+	suit = /obj/item/clothing/suit/armor/fallout/legion/veteran
+	gloves = /obj/item/clothing/gloves/fallout/legion/darkwraps
+	mask = /obj/item/clothing/mask/fallout/legion/clothmask
+	suit_store = /obj/item/gun/ballistic/automatic/pistol/fallout/m9mm
```

Comment:
9mm pistol for the Vet Legionary is also fairly cringe. Weapon choice is definitely going to need to be discussed a lot.